assets/stylesheets/coderay.css

```diff
@@ -46,6 +46,11 @@ a span.icon>.fa{cursor:inherit;}
         min-height: 349px;
     }
 }
+
+.grid-item a,
+.guide-list .guide .guide-title {
+    text-decoration: none !important;
```

Comment:
This removes the underline of guides title.

Currently: 

![CleanShot 2024-10-04 at 09 51 33@2x](https://github.com/user-attachments/assets/096333b2-a9d0-47eb-8536-a956dfda7793)

with this PR: 

![CleanShot 2024-10-04 at 09 52 06@2x](https://github.com/user-attachments/assets/c8885634-c60b-4313-92e4-ea04b74ff96b)